source/parser.hera

```diff
@@ -524,8 +524,8 @@ ConditionalExpression
 TernaryRest
   NestedTernaryRest
   # NOTE: Ternary `a ? b : c` is disabled if CoffeeScript binary existential `a ? b` is enabled
-  !CoffeeBinaryExistentialEnabled &" " _? QuestionMark ExtendedExpression __ Colon ExtendedExpression ->
-    return $0.slice(2)
+  !CoffeeBinaryExistentialEnabled _ QuestionMark ExtendedExpression __ Colon ExtendedExpression ->
```

Comment:
I'm slightly worried about the loss of `&" "`. Is an inline conment here sufficient? Maybe...